tools/api-markdown-documenter/src/api-item-transforms/configuration/DocumentationSuiteOptions.ts

```diff
@@ -91,6 +91,10 @@ export type HierarchyBoundaries = ApiMemberKind[];
 /**
  * Options for configuring the documentation suite generated by the API Item -\> Documentation Domain transformation.
  *
+ * @remarks
+ * Used as API input.
+ * Optional properties will be filled in with default values if not provided.
+ *
  * @public
  */
 export interface DocumentationSuiteOptions {
```

Comment:
TODO: didn't mean to bundle these changes here. Will revert them for now, and make the appropriate changes in a follow-up PR.